sdk/react-native-expo/project.json

```diff
@@ -45,7 +45,8 @@
                         "input": ".",
                         "output": "."
                     }
-                ]
+                ],
+                "updateBuildableProjectDepsInPackageJson": true
```

Comment:
@ajwootto interesting that the migration added these, will need to test this more later.

Reply:
just because it was defaulted to true before, and now it defaults false